<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ParametersWidget</class>
 <widget class="QWidget" name="ParametersWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>105</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Parameters</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="2" >
    <widget class="QRadioButton" name="disableParameters" >
     <property name="text" >
      <string>Disable</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QRadioButton" name="parametersList" >
     <property name="toolTip" >
      <string>read parameter values from a list</string>
     </property>
     <property name="whatsThis" >
      <string>Check this to let KmPlot use parameter values given in a list. Edit this list by clicking the button on the right.</string>
     </property>
     <property name="text" >
      <string>From a list:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="KPushButton" name="editParameterListButton" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="toolTip" >
      <string>Edit the list of parameters</string>
     </property>
     <property name="whatsThis" >
      <string>Click here to open a list of parameter values. Here you can add, remove, and change them.</string>
     </property>
     <property name="text" >
      <string>Edit List...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QComboBox" name="listOfSliders" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="toolTip" >
      <string>Select a slider</string>
     </property>
     <property name="whatsThis" >
      <string>Select one of the sliders to change the parameter value dynamically. The values vary from 0 (left) to 100 (right).</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QRadioButton" name="parameterSlider" >
     <property name="toolTip" >
      <string>change parameter value by moving a slider</string>
     </property>
     <property name="whatsThis" >
      <string>Check this to change the parameter value by moving a slider. Select the slider from the list box on the right. The values vary from 0 (left) to 100 (right).</string>
     </property>
     <property name="text" >
      <string>Use a slider:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends></extends>
   <header>kpushbutton.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>disableParameters</tabstop>
  <tabstop>parametersList</tabstop>
  <tabstop>editParameterListButton</tabstop>
  <tabstop>parameterSlider</tabstop>
  <tabstop>listOfSliders</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>parametersList</sender>
   <signal>toggled(bool)</signal>
   <receiver>editParameterListButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>76</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel" >
     <x>239</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>parameterSlider</sender>
   <signal>toggled(bool)</signal>
   <receiver>listOfSliders</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>116</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel" >
     <x>234</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
